front/ui/ui-charts/src/manchette/hooks/useManchetteWithSpaceTimeChart.tsx

```diff
@@ -204,8 +214,20 @@ const useManchetteWithSpaceTimeChart = ({
               maxZoomMillimeterPerPx,
               newSpaceScale
             );
-            const topRectSide = Math.min(prev.rect.spaceStart, prev.rect.spaceEnd);
-            newYOffset = Math.abs(spaceOrigin - topRectSide) / newSpaceScale;
+            if (newSpaceScale !== maxZoomMillimeterPerPx) {
+              const topRectSide = Math.min(prev.rect.spaceStart, prev.rect.spaceEnd);
+              newYOffset = Math.abs(spaceOrigin - topRectSide) / newSpaceScale;
+            } else {
+              newYOffset = Math.abs(
+                sideOffset(
+                  spaceOrigin,
+                  newSpaceScale,
+                  prev.rect.spaceStart,
+                  prev.rect.spaceEnd,
+                  height
+                )
+              );
+            }
```

Comment:
Why not directly use sideOffset ?
```suggestion
            newYOffset = Math.abs(
              sideOffset(
                spaceOrigin,
                newSpaceScale,
                prev.rect.spaceStart,
                prev.rect.spaceEnd,
                height
              )
            );
```

Reply:
I had to adapt the sideOffset computation a bit with the padding, but i made the change